### ISSUE TITLE
rpc: node_info may return nil on start

### ISSUE THREAD

**Issue by melekes** [tendermint/tendermint#1199]
```
When we send a status rpc request via websocket immediately after establishing the websocket connection, node info will may be null while later calls will have a non null node info.

Delaying the call 5+ seconds yields expected results.

Looks like the other reactors are not fully up but rpc is already allowed ?
Does that male sense?

PS tendermint 0.12
```

Need to check / try to reproduce and fix.

**Comment by srmo** [comment]
wanted to add the following:
- the Websocket connection is done by the ABCI app in a thread separated from the actual ABCI component
- this happens when it is the first connection between ABCI App and tendermint is established, i.e. consensus connection and websocket are a first connection
- didn't test if this also happens on websocket reconnects after consensus was already established

**Comment by ebuchman** [comment]
Can you check if this is solved in #1511 ?
